feature-libs/asm/root/model/bind-cart.model.ts

```diff
@@ -0,0 +1,4 @@
+export interface BindCartOptions {
```

Comment:
Probably this model should be added to the existing model file.

Reply:
The existing model file I referred to is `feature-libs/asm/core/models/asm.models.ts`

However, the models should really be in the root entry point.  You can leave your model here and instead we should bring over the other ones at some point.